src/configuration.cpp

```diff
@@ -451,20 +453,80 @@ chimera::CompiledConfiguration::CompiledConfiguration(
     // Override individual templates if specified in the configuration.
     if (bindingNode_)
     {
-        if (const YAML::Node classTemplateNode = bindingNode_["class"])
-            bindingDefinition_.class_cpp = Lookup(classTemplateNode);
+        if (const auto node = lookupYAMLNode(bindingNode_, "class"))
+        {
+            if (node.IsScalar())
+            {
+                bindingDefinition_.class_cpp = Lookup(node);
+            }
+            else
+            {
+                if (const auto header_node = lookupYAMLNode(node, "header"))
+                    bindingDefinition_.class_h = Lookup(header_node);
+                if (const auto source_node = lookupYAMLNode(node, "source"))
+                    bindingDefinition_.class_cpp = Lookup(source_node);
+            }
```

Comment:
With the sentinel value this would look more like:
```suggestion
            if (node.IsNull())  // This needs to be different than *undefined*, which means unspecified
            {
                bindingDefinition_.class_h = chimera::util::FLAG_NO_RENDER;
                bindingDefinition_.class_cpp = chimera::util::FLAG_NO_RENDER;
            }
            if (node.IsScalar())
            {
                bindingDefinition_.class_h = chimera::util::FLAG_NO_RENDER;
                bindingDefinition_.class_cpp = Lookup(node);
            }
            else
            {
                if (const auto header_node = lookupYAMLNode(node, "header"))
                    bindingDefinition_.class_h = Lookup(header_node);
                if (const auto source_node = lookupYAMLNode(node, "source"))
                    bindingDefinition_.class_cpp = Lookup(source_node);
            }
```

Reply:
Done https://github.com/personalrobotics/chimera/pull/289/commits/3fb77d63aa41ac4da2e2680fb509ed101dc6a03a